bin/cli.js

```diff
@@ -2,12 +2,83 @@
 
 'use strict';
 
+const commandLineArgs = require('command-line-args');
+const commandLineUsage = require('command-line-usage');
 const readline = require('readline');
 
 const Cli = require('../lib/cli/cli');
 
 let rl = null;
 
+const options = [
+  {
+    name: 'verbose',
+    alias: 'v',
+    type: Boolean,
+    multiple: true,
+    description: 'Verbosely print incoming and outgoing messages',
+  },
+  {
+    name: 'no-reconnect',
+    alias: 'R',
+    type: Boolean,
+    default: false,
+    description: 'Disable reconnection',
+  },
+  {
+    name: 'heartbeat-interval',
+    alias: 'i',
+    type: Number,
+    default: 0,
```

Comment:
The correct name of this property seems to be `defaultValue`.
Although I am not quite sure if we even need to use this property anywhere.